docs/index.html

```diff
@@ -0,0 +1,252 @@
+
+            <li class="hover:text-rose-700">
+              <a href="#sponsor">Sponsor</a>
+            </li>
+
+            <li class="hover:text-rose-700">
+              <a href="#news">News</a>
+            </li>
+          </ul>
+        </nav>
+
+        <a href="#home">
+          <h3 class="border-4 px-4 py-2 border-rose-700 text-rose-700 font-bold hover:bg-rose-700 hover:text-white hover:shadow-md hover:shadow-zinc-500 ease-in-out delay-200">
+            Garden Campaign
+          </h3>
+        </a>
+      </div>
+
+      <!-- Mobile Menu -->
+
+      <div class="md:hidden bg-hero-image bg-center bg-no-repeat bg-cover bg-blend-overlay bg-stone-300 px-5 fixed top-0 left-0 right-0 h-[100vh] transition translate-x-full delay-100 target:-translate-x-0" id="menu">
+        <div class="md:hidden flex justify-between content-center flex-wrap pb-[50px] py-3">
+          <a href="#home">
+            <img src="./images/flower-logo1.png" alt="Logo" width="150px" class="block">
+          </a>
+
+          <a href="#home">
+            <i class="fa-solid fa-x fa-xl"></i>
+          </a>
+        </div>
+
+        <nav>
+          <ul class="flex flex-col gap-5 text-3xl font-bold px-6">
+            <li>
+              <a href="index.html">Home</a>
+            </li>
+
+            <li>
+              <a href="about.html">About</a>
+            </li>
+
+            <li>
+              <a href="#program">Program</a>
+            </li>
+
+            <li>
+              <a href="#join">Join</a>
+            </li>
+
+            <li>
+              <a href="#sponsor">Sponsor</a>
+            </li>
+
+            <li>
+              <a href="#news">News</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <!-- Hero Section -->
+
+      <section class="bg-hero-image bg-center bg-no-repeat bg-cover bg-blend-overlay bg-stone-300 pb-[100px] px-4 py-3 md:px-[5%] lg:px-[15%] md:py-[100px]">
+
+        <div class="md:hidden flex justify-between content-center flex-wrap pb-[50px]">
+          <a href="#home">
+            <img src="./images/flower-logo1.png" alt="Logo" width="150px" class="block">
+          </a>
+
+          <a href="#menu">
+            <i class="fa-solid fa-bars fa-xl"></i>
+          </a>
+        </div>
+
+        <h2 class="text-3xl text-emerald-800 mb-2">
+          "Hello, Sharing world!"
+        </h2>
+
+        <h1 class="text-6xl font-extrabold text-emerald-800/[0.8] mb-6 lg:w-[70%] md:text-8xl">
+          Garden and Flower Show 2023
+        </h1>
+
+        <p class="text border-4 border-zinc-500 p-4 mb-4 md:w-[70%] lg:w-[55%] md:mb-8">
+          A joyful celebration of amazing garden designs, gorgeous floral displays and exclusive shopping deals.
+        </p>
+
+        <h2 class="text-4xl font-extrabold mb-2 md:mb-4">
+          2023.05.15. ~ 05.20.
+        </h2>
+        <h3 class="text-xl">
+          @ Hungary, Tiszakürti Arborétum
+        </h3>
+      </section>
```

Comment:
Kindly update the hero section in the home page to match the behance design better🙏

Your design:
<img width="507" alt="image" src="https://user-images.githubusercontent.com/37116322/233504503-b7bbd015-a198-46e7-bdbf-9a465399265c.png">

Behance design
<img width="626" alt="image" src="https://user-images.githubusercontent.com/37116322/233504557-88d34658-905b-4bcb-92a0-83d321281c78.png">

A few things to note:
- Your project is really nice. But the project requirement state that you should stick to the design guidelines regarding colors. So please change the color of the main title and subtitle in this section to be `#f3401e`